packages/dds/migration-shim/README.md

```diff
@@ -0,0 +1,61 @@
+# @fluid-experimental/migration-shim
```

Comment:
This looks like an unrelated change. Probably from a merge. This package was deleted, so you can just delete this file.